docs/content/docs/operators/classification/knn.md

```diff
@@ -175,15 +176,10 @@ predict_data = t_env.from_data_stream(
 knn = KNN()
 model = knn.fit(train_data)
 output = model.transform(predict_data)[0]
-output.execute().print()
+print([result for result in t_env.to_data_stream(output).execute_and_collect()])
 
 # output
-# +----+--------------------------------+--------------------------------+--------------------------------+
-# | op |                       features |                          label |                     prediction |
-# +----+--------------------------------+--------------------------------+--------------------------------+
-# | +I |                     [4.0, 4.1] |                            5.0 |                            5.0 |
-# | +I |                  [300.0, 42.0] |                            2.0 |                            2.0 |
-# +----+--------------------------------+--------------------------------+--------------------------------+
+# [<Row(DenseVector([4.0, 4.1]), 5.0, 5.0)>, <Row(DenseVector([300.0, 42.0]), 2.0, 2.0)>]
```

Comment:
Would it be better if we reformat the output to improve its readability? For example, the KMeans Java example would print the results as follows.
```java
for (CloseableIterator<Row> it = output.execute().collect(); it.hasNext(); ) {
    Row row = it.next();
    DenseVector vector = (DenseVector) row.getField(featuresCol);
    int clusterId = (Integer) row.getField(predictionCol);
    System.out.println("Vector: " + vector + "\tCluster ID: " + clusterId);
}